api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2895,4 +2897,22 @@ service AuthService {
 
   // UpdateHeadlessAuthenticationState is a request to update a headless authentication's state.
   rpc UpdateHeadlessAuthenticationState(UpdateHeadlessAuthenticationStateRequest) returns (google.protobuf.Empty);
+
+  // CreatePlugin creates a plugin resource.
```

Comment:
This + implmentation in auth essentially duplicates the plugin *sub-service*:
* Proto https://github.com/gravitational/teleport/tree/master/api/proto/teleport/plugins/v1
* Implementation in `e` https://github.com/gravitational/teleport.e/tree/master/lib/plugins/pluginsv1

Could we simply move the existing service into OSS instead?

Reply:
Would it make sense to first add the duplicated code in this PR then make the changes to /e and following that?